webots_ros2_core/webots_ros2_core/webots_differential_drive_node.py

```diff
@@ -111,8 +113,8 @@ def step(self, ms):
             return
 
         # Calculate velocities
-        v_left_rad = (left_wheel_ticks - self._prev_left_wheel_ticks) / time_diff_s
-        v_right_rad = (right_wheel_ticks - self._prev_right_wheel_ticks) / time_diff_s
+        v_left_rad = (left_wheel_ticks - self._prev_left_wheel_ticks) / time_diff_s if time_diff_s > 0.0 else 0.0
```

Comment:
These changes are not required any more as of:
```Python
if time_diff_s == 0.0:
            return
```
declared above. `turtlebot3_cartographer` didn't like two odometry messages to be published with the same timestamp.

Reply:
Well spotted, this is now fixed.